src/components/select-field/BaseSelectField.js

```diff
@@ -85,6 +85,24 @@ class BaseSelectField extends React.Component<Props, State> {
             isOpen: false,
             shouldScrollIntoView: false,
         };
+
+        this.fieldRef = React.createRef();
+    }
+
+    componentDidMount() {
+        this.calculateOffsetTopValue();
+    }
+
+    /**
+     * Calculate this field's distance from the top.
+     * This allows us to create an overflow container for the field's dropdown menu that is navigable using both
+     * the mouse and arrow keys and will always fit within the current viewport.
+     */
+    calculateOffsetTopValue() {
+        this.offsetTopValue =
+            window.pageYOffset +
+            this.fieldRef.current.getBoundingClientRect().top +
```

Comment:
We should be checking to make sure that `this.fieldRef.current` exists before operating on it. You can use destructured renaming to do so:

```
const { current: fieldRef } = this.fieldRef;

if (!fieldRef) {
    return 0;
}

return window.pageYOffset + fieldRef.getBoundingClientRect().top + 75;
```